Rakefile

```diff
@@ -159,6 +159,14 @@ namespace :docs do
       Primer::UnderlineNavComponent
     ]
 
+    js_components = [
```

Comment:
I wonder if we could make this declaration live on the component itself similar to status. I think that way it's harder to forget about this file ;) WDYT?

Reply:
I think that's a good followup! wanted to keep it simple for now